src/app/admin/services/index.ts

```diff
@@ -1,2 +1,3 @@
 export { ApiService as AdminApi } from './api/api.service';
 export { ControllerService as AdminController } from './controller/controller.service';
+export { UserControllerService as UserController } from './user-controller/user-controller.service';
```

Comment:
do you think we neeed rename in here? I wuld prefere have name UserControllerService.  I think it will make things easier in the future.